src/app/feature/consultant/components/consultant-experience/consultant-experience.component.ts

```diff
@@ -78,6 +80,7 @@ export class ConsultantExperienceComponent implements OnInit {
 
     updatedExperience.splice(selectedIndex, 1);
 
+    this.notification.openDeletedExperienceSnackBar();
```

Comment:
Please put this in the subscription and add a catchError pipe with error message handling. Otherwise the success message will still display even if the update fails